### DISCUSSION TITLE
Installing tool dependencies (metapackage support)

### DISCUSSION THREAD

**Discussion by jnm2** [cake-build/cake#3037] [Q&A]
##### @mholo65 https://github.com/cake-build/cake/issues/2318#issuecomment-425649760:

> @ChrisMaddock @jnm2, regarding metapackages and `#tool`, feel free to open another issue to discuss it further. I think it would be a good idea to support, hence here’s the [code](https://github.com/cake-build/cake/blob/develop/src/Cake.NuGet/Install/NuGetPackageInstaller.cs#L150) in case anyone needs a PR for hacktoberfest 😉

Are there any downsides to installing all tool dependencies?

**Comment by devlead** [comment]
> Are there any downsides to installing all tool dependencies?

No, we just need to make sure tools are resolved in all downloaded packages.
